src/index.js

```diff
@@ -1,49 +1,23 @@
 /* eslint-disable quotes */
```

Comment:
- [ ] [optional] @francis3400 its not good practise to disable linters, always try and solve them.